ui/src/AudioRecorder.js

```diff
@@ -0,0 +1,36 @@
+export class AudioRecorder {
+
+    constructor() {
+        this.recordingUrl = "";
+        this.mediaRecorder = "";
+        this.recordedChunks = [];
+    }
+
+    startRecording() {
+        navigator.mediaDevices.getUserMedia({ audio: true, video: false })
+            .then(function(stream) {
+
+            this.mediaRecorder = new MediaRecorder(stream);
+            this.mediaRecorder.start();
+
+        });
```

Comment:
Can you handle the error or add a TODO to handle the error in case getUserMedia fails? For now, it can be a simple alert. In the future, you would have to decide how to handle cases like if the user denies the permission, what would the UI do?